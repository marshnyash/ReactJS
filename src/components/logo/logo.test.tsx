import { render } from '@testing-library/react';
import React from 'react';

import Logo from './logo';

describe("Logo", () => {
  test("renders Logo snapshot", () => {
    const { asFragment } = render(<Logo />);
    expect(asFragment()).toMatchSnapshot();
  });
});
