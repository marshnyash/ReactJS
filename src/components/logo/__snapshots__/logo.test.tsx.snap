// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders Logo snapshot 1`] = `
<DocumentFragment>
  <span
    class="Logo-sc-1jha6qf-0 heMtbf"
  >
    netflix
  </span>
</DocumentFragment>
`;
